docs/user_guide/nvflare_cli/job_cli.rst

```diff
@@ -208,3 +208,40 @@ and change the app_1 batch_size to 4, app_2 batch_size to 6 for sag_pt_deploy_ma
 
     The app names must be defined in the job template being used: in this case ``app_1``, ``app_2``, and ``app_server``,
     are in ``sag_pt_deploy_map``.
+
+***************************
+FLARE Job Template Registry
+***************************
+
+Below is a table of all available :github_nvflare_link:`Job Templates <job_templates>`.
+
+.. csv-table::
+    :header: Example,Client Category,Controller-Type,Description
```

Comment:
For the name "Client Category", I change the section "FL Client" -> "Task Execution", because I think our options (Executors VS ModelLearner VS ClientAPI) are really related to how task being executed.

I think the "FL Client" is referring to a larger concept including communication (pull_task, push_results), start worker process (ProcessExecutor), init components and some filters and fire events. 

So suggest changing "Client Category" to maybe "Task Execution Category" or "Task Execution"? Or do you have some other ideas?

Reply:
changed to "Execution API Type"